lib/src/widgets/message_container.dart

```diff
@@ -106,6 +117,17 @@ class MessageContainer extends StatelessWidget {
                     image: message.image,
                   ),
                 ),
+            if(messageButtonsBuilder != null)
```

Comment:
I think if the buttons property is set it should be given priority over `messageButtonsBuilder` since it is global and will be set for all messages and `buttons` is message specific.